independent-publisher-connectors/Workable/apiProperties.json

```diff
@@ -0,0 +1,61 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Workable API Key",
+          "description": "Provide your API Key from Workable",
+          "tooltip": "Provide your API Key from Workable",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      },
+        "Workable API endpoint": {
```

Comment:
@DavidBreathe can you please explain why you have this property? is it possible to use camel case?

Reply:
The property is needed for defining the API subdomain for API endpoint <subdomain>.workable.com which is different for all customers / clients using the service.